src/cryptography/hazmat/bindings/openssl/engine.py

```diff
@@ -145,11 +147,23 @@
 const ENGINE_CMD_DEFN *ENGINE_get_cmd_defns(const ENGINE *);
 EVP_PKEY *ENGINE_load_private_key(ENGINE *, const char *, UI_METHOD *, void *);
 EVP_PKEY *ENGINE_load_public_key(ENGINE *, const char *, UI_METHOD *, void *);
+
 void ENGINE_add_conf_module(void);
 """
 
 MACROS = """
 void ENGINE_load_cryptodev(void);
+
+/* Engine's client cert functions were added in OpenSSL 0.9.8i */
+int ENGINE_set_load_ssl_client_cert_function(ENGINE *,
+                                             ENGINE_SSL_CLIENT_CERT_PTR);
+ENGINE_SSL_CLIENT_CERT_PTR ENGINE_get_ssl_client_cert_function(
+    const ENGINE *);
+int ENGINE_load_ssl_client_cert(ENGINE *, SSL *,
+                                Cryptography_STACK_OF_X509_NAME *,
+                                X509 **, EVP_PKEY **,
+                                Cryptography_STACK_OF_X509 **,
+                                UI_METHOD *, void *);
```

Comment:
This isn't right, as far as I can tell the signature for this function changed at some point over the years.